specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -179,12 +190,13 @@ function TreeView<SCHEMA extends AnyTree>({
         />
         <Button.Icon
           aria-pressed={isSplit}
+          disabled={screenWidth <= 640}
```

Comment:
create a variable `canSplit` and use that, rather than duplicating the magic number 3 times